org.eclipse.scanning.example/src/org/eclipse/scanning/example/malcolm/DummyMalcolmRecord.java

```diff
@@ -0,0 +1,453 @@
+    			add("writeable", ScalarType.pvBoolean).
+    			add("label", ScalarType.pvString).
+    			setId("malcolm:core/NumberMeta:1.0").
+    			createStructure();
+        
+        Structure floatNumberMetaStructure = fieldCreate.createFieldBuilder().
+    			add("dtype", ScalarType.pvString).
+    			add("description", ScalarType.pvString).
+    			addArray("tags", ScalarType.pvString).
+    			add("writeable", ScalarType.pvBoolean).
+    			add("label", ScalarType.pvString).
+    			setId("malcolm:core/NumberMeta:1.0").
+    			createStructure();
+        
+        Structure stringArrayMetaStructure = fieldCreate.createFieldBuilder().
+    			add("description", ScalarType.pvString).
+    			addArray("tags", ScalarType.pvString).
+    			add("writeable", ScalarType.pvBoolean).
+    			add("label", ScalarType.pvString).
+    			setId("malcolm:core/StringArrayMeta:1.0").
+    			createStructure();
+        
+        Structure numberArrayMetaStructure = fieldCreate.createFieldBuilder().
+    			add("dtype", ScalarType.pvString).
+    			add("description", ScalarType.pvString).
+    			addArray("tags", ScalarType.pvString).
+    			add("writeable", ScalarType.pvBoolean).
+    			add("label", ScalarType.pvString).
+    			setId("malcolm:core/NumberArrayMeta:1.0").
+    			createStructure();
+
+        Structure mapMetaStructure = fieldCreate.createFieldBuilder().
+    			add("description", ScalarType.pvString).
+    			addArray("tags", ScalarType.pvString).
+    			add("writeable", ScalarType.pvBoolean).
+    			add("label", ScalarType.pvString).
+    			addArray("required", ScalarType.pvString).
+    			setId("malcolm:core/MapMeta:1.0").
+    			createStructure();
+        
+        Structure tableElementsStructure = fieldCreate.createFieldBuilder().
+    			add("detector", stringArrayMetaStructure).
+    			add("filename", stringArrayMetaStructure).
+    			add("dataset", stringArrayMetaStructure).
+    			add("users", numberArrayMetaStructure).
+    			createStructure();
+        
+        Structure tableMetaStructure = fieldCreate.createFieldBuilder().
+    			add("elements", tableElementsStructure).
+    			add("description", ScalarType.pvString).
+    			addArray("tags", ScalarType.pvString).
+    			add("writeable", ScalarType.pvBoolean).
+    			add("label", ScalarType.pvString).
+    			setId("malcolm:core/TableMeta:1.0").
+    			createStructure();
+        
+        Structure pointGeneratorMetaStructure = fieldCreate.createFieldBuilder().
+    			add("description", ScalarType.pvString).
+    			addArray("tags", ScalarType.pvString).
+    			add("writeable", ScalarType.pvBoolean).
+    			add("label", ScalarType.pvString).
+    			setId("malcolm:core/PointGeneratorMeta:1.0").
+    			createStructure();
+        
+        // Attributes
+        Structure choiceStructure = fieldCreate.createFieldBuilder().
+    			add("meta", choiceMetaStructure).
+    			add("value", ScalarType.pvString).
+    			setId("epics:nt/NTScalar:1.0").

```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "epics:nt/NTScalar:1.0" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)